Palindrome.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.Scanner;
+
+public class Palindrome {
+	public static void main(String[] args) {
+		Scanner input = new Scanner(System.in);
+		int inputNumber;
+		int upperLimit = 1000000000;
+		int arraySize = 0;
+		int[] numbersArray;
+		boolean numberValid = true;
+		
+		System.out.print("Enter your number: ");
+		inputNumber = input.nextInt();
+		while (inputNumber < 0 || inputNumber > upperLimit) {
+			System.out.println("You can only enter value from 0 to " + upperLimit);
+			inputNumber = input.nextInt();
+		}
+		
+		for (int divisor = upperLimit; divisor > 1; divisor /= 10) {
+			int currentNumber = (int) ((inputNumber % divisor) / (divisor / 10));
+			if (currentNumber != 0) {
+				arraySize++;
+			} else {
+				upperLimit /= 10;
+			}
+		}
+		
```

Comment:
there's an easier way to find the arraySize - do you know how ?

Reply:
I guess I do now - divide input by 10 while it's > 0, right?

Reply:
👍